src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificatesTabCoordinator.swift

```diff
@@ -199,8 +200,89 @@ final class HealthCertificatesTabCoordinator {
 		viewController.present(navigationController, animated: true)
 	}
 	
+	
 	private func showAdmissionScenarios() {
-		// to.do show admission scenarios list - EXPOSUREAPP-11764
+		let result = self.cclService.dccAdmissionCheckScenarios()
+		switch result {
+		case .success(let scenarios):
+			self.store.dccAdmissionCheckScenarios = scenarios
+			let listItems = scenarios.scenarioSelection.items.map({
+				SelectableValue(
+					title: $0.titleText.localized(cclService: cclService),
+					subtitle: $0.subtitleText?.localized(cclService: cclService),
+					identifier: $0.identifier,
+					isEnabled: $0.enabled
+				)
+			})
+			let selectValueViewModel = SelectValueViewModel(
+				listItems,
+				presorted: true,
+				title: scenarios.scenarioSelection.titleText.localized(cclService: cclService),
+				preselected: nil,
+				initialValue: nil,
+				accessibilityIdentifier: AccessibilityIdentifiers.LocalStatistics.selectState,
+				selectionCellIconType: .none
+			)
+			selectValueViewModel.$selectedValue.sink { [weak self] federalState in
+				guard let self = self, let state = federalState else {
+					return
+				}
+				self.healthCertificateService.lastSelectedScenarioIdentifier = state.identifier
+				self.showActivityIndicator(from: self.viewController.view)
+				self.healthCertificateService.updateDCCWalletInfosIfNeeded(
+					isForced: true
+				) { [weak self] in
+					self?.hideActivityIndicator()
+					DispatchQueue.main.async {
+						self?.viewController.presentedViewController?.dismiss(animated: true, completion: nil)
+					}
+				}
+			}.store(in: &subscriptions)
+			
+			let selectValueViewController = SelectValueTableViewController(
+				selectValueViewModel,
+				closeOnSelection: false,
+				dismiss: { [weak self] in
+					self?.viewController.presentedViewController?.dismiss(animated: true, completion: nil)
+				})
+			self.viewController.present(
+				UINavigationController(rootViewController: selectValueViewController),
+				animated: true
+			)
+		case .failure(let error):
+			showErrorAlert(title: AppStrings.HealthCertificate.Error.title, error: error)
+			Log.error(error.localizedDescription)
+		}
+	}
+	private let activityIndicatorView = QRScannerActivityIndicatorView(frame: .zero, title: AppStrings.HealthCertificate.Overview.loadingIndicatorLabel)
```

Comment:
Usually were putting `let`s together, then `var`s and then `func`s, so I would suggest to move this up to the other `let`s